lib/secure_headers/headers/cookie.rb

```diff
@@ -16,6 +16,11 @@ def validate_config!(config)
 
     def initialize(cookie, config)
       @raw_cookie = cookie
+      unless config == SecureHeaders::OPT_OUT
+        config ||= {}
```

Comment:
I wonder if it would make sense to define the default as `{ secure: true, httponly: true}` as opposed to checking for `nil` in the configuration.

I'm of the opinion that if a configuration is defined (even if it disables `secure` or `httponly`), you shouldn't have to opt-out of each configuration explicitly.